app/services/project_data_service.rb

```diff
@@ -107,4 +111,17 @@ def fetch_github_file(path)
   rescue Octokit::NotFound
     # file not found - don't fail if ruby project doesn't have node etc
   end
+
+  def report_runtime_version_status(dependency)
+    Horizon.dogstatsd.gauge(
+      'runtime.version_status', # Metric name
+      dependency.update_required? ? -1 : 1, # The value associated with the metric. -1 = out of date, 1 = up to date
+      tags: [
+        "runtime:#{dependency.name}",
+        "project:#{@project.name}",
+        "criticality:#{@project.criticality}",
+        "team:#{@project.tags&.join(':')}"
```

Comment:
We currently accept `nil` team values. This seems reasonable, though ends up looking kind of weird in datadog, with a tag of just `team:`. Happy to hear alternatives here.

Reply:
I also wonder if the tag names are too generic and should be prepended with `horizon`?

Reply:
> We currently accept nil team values. This seems reasonable, though ends up looking kind of weird in datadog, with a tag of just team:. Happy to hear alternatives here.

What if we do not include the `team:` tag in the list when a project has no team value? Or set `none` or something similar?

---

> I also wonder if the tag names are too generic and should be prepended with horizon?

So it would be?:
```
horizon-runtime:ruby
horizon-project:eigen
...
```

I suppose if it's not obvious that these are coming from Horizon it may be helpful but it doesn't seem necessary.

Reply:
> Or set none or something similar?

Good call! I think this is probably the best option and matches what does when it tries to set a tag with an unknown value (e.g. `env:none` when developing locally). 

> I suppose if it's not obvious that these are coming from Horizon it may be helpful but it doesn't seem necessary.

Yeah, that's what I'm thinking. We will be able to filter by `service:horizon` to always filter by `horizon`.

<img width="1178" alt="Screenshot 2023-06-29 at 9 23 59 AM" src="https://github.com/artsy/horizon/assets/38149304/ab551e55-9bd3-4b16-a2a0-90c7ee52038d">

I'll keep it as is for now and we can reconsider if it becomes confusing or problematic.